extensions/wikia/CommunityPage/styles/components/_variables.scss

```diff
@@ -8,6 +8,8 @@ $community-page-font-family: "HelveticaNeue", Helvetica, Arial, sans-serif;
 $module-default-line-height: 20px;
 $avatar-size: 30px;
 $avatar-flex: 0 0 $avatar-size;
+$username-column-width: 174px;
+$userrank-column-width: 26px;
```

Comment:
Can't you just use two different variables for those? What's the reason behind sharing those? Is `userrank-column-width` use anywhere else?